docs/projects/zenfs.md

```diff
@@ -40,49 +37,61 @@ Using *ZenFS*, data garbage collection is performed only by RocksDB with the
 LSM-tree table compaction process. There is no garbage collection executed
 by ZenFS, nor by the ZNS device controller.
 
-## Prerequisites
+!!! Note
+     Further information is available in the <a href="https://www.usenix.org/conference/atc21/presentation/bjorling" target="_blank">ZNS: Avoiding the Block Interface Tax for Flash-based SSDs</a> USENIX ATC 2021 article.
+
+## Getting Started
+
+### Prerequisites
 
-*ZenFS* requires a Linux kernel implementing support for NVMe Zoned namespaces,
-that is, a kernel version 5.9 or newer. The kernel used must also be configured
-with [zoned block device support enabled](/linux/config#kernel-configuration).
+*ZenFS* requires Linux kernel 5.9 or newer and the kernel used must also be
+configured with [zoned block device support enabled](/linux/config#kernel-configuration).
 
-*ZenFS* code uses the [*libzbd*](/projects/libzbd) library. The latest
-version of this library must be compiled and installed prior to compiling,
-installing and using *RocksDB-ZenFS*.
+*ZenFS* utilizes the [*libzbd*](/projects/libzbd) library. The latest
+version of this library must be compiled and installed prior to building and
+installing.
 
-## Building and Installing ZenFS
+### Building and Installing ZenFS
 
-Detailed instructions on how to compile and install *ZenFS* is provided by the
-*ZenFS* project <a href="https://github.com/westerndigitalcorporation/zenfs/blob/master/README.md" target="_blank">
+*ZenFS* is embedded into RocksDB, and is available as a submodule in RocksDB,
+which must be explicitly enabled when compiling RocksDB.
+
+Instructions on how to compile and install RocksDB with *ZenFS* are maintained
+in the *ZenFS* project <a href="https://github.com/westerndigitalcorporation/zenfs/blob/master/README.md" target="_blank">
 README file</a>.
 
-As mentioned in the *ZenFS* build instructions, it is also important to set
-the IO scheduler of the NVMe ZNS device to *mq-deadline* to avoid write
-operations from being reordered. This can be automatically done on system boot
-using a [*udev* rule](/linux/sched#automatic-persistent-configuration).
+!!! NOTE
+     Remember to set the block device IO scheduler to deadline to avoid write
```

Comment:
s/avoid/prevent/

I am astonished at the near-perfect use of English in this document, and delighted that I read so long before making a suggestion.